hypervisor/src/hypervisor/partition.rs

```diff
@@ -507,7 +507,7 @@ impl Base {
 
 #[derive(Debug)]
 pub(crate) struct Partition {
-    base: Base,
+    pub(crate) base: Base,
```

Comment:
This exposes way more than required. Better implement `Partition::name()`.

Reply:
Agreed! Will do implement your suggestion

Reply:
Done